apparelo/apparelo/doctype/item_production_detail/item_production_detail.py

```diff
@@ -223,7 +226,7 @@ def create_process_details(self):
 				if process.input_item:
 					pass
 				if process.ipd_name:
-    					if process.input_index:
+					if process.input_index:
```

Comment:
You did some indentation change here, yesterday, Is this conflicting or removing that?